python/kernmlops/data_collection/bpf_instrumentation/madvise.py

```diff
@@ -97,7 +97,7 @@ def _madvise_eh(self, cpu, madvise_struct, size):
         MadviseStat(
           tgid=event.tgid,
           ts_ns=event.ts_ns,
-          address=event.address,
+          address=hex(event.address),
```

Comment:
We want to leave this as an int, the only reason to convert is for display purposes, and we leave that for other to do. For importing this into code that might produce a model it seems like an unnecessary conversion that could pollute the resulting parquet files with unstructured data.